packages/core/src/store/mad-config/hooks.ts

```diff
@@ -56,7 +57,13 @@ export const useAbout = () => {
 export const useServiceNow = () => {
     const config = useMadConfig();
     return config.serviceNow;
-}
+};
+
+export const useNavigateToMainRoute = () => {
+    const config = useMadConfig();
+    const navigation = useNavigation();
```

Comment:
Should `useCoreStackNavigation` be used instead?

Reply:
Using `useCoreStackNavigation` will lead to ts errors because `CoreStackParamList` is not the same type as `object`. `navigateToMainRouteFn` will never navigate to any routes in the `CoreStackParamList` anyway